src/app/clusters/operational-credentials-server/operational-credentials-server.cpp

```diff
@@ -30,6 +30,7 @@
 #include <app/CommandHandler.h>
 #include <app/ConcreteCommandPath.h>
 #include <app/EventLogging.h>
+#include <app/InteractionModelEngine.h>
```

Comment:
Why?

Reply:
See calls to `InteractionModel::GetInstance()->...` in the file.